docs/guides/gradle.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Gradle configuration
+headline: Documentation
+bodyclass: docs
+layout: docs
+---
+
+# Configuring a project with Gradle
+
+Spine provides build-time tools for code generation and analysis. These tools are implemented
+as [Gradle plugins](https://docs.gradle.org/current/userguide/plugins.html) — an extendable, uniform
+way to tap into a project build process.
+
+The minimal Gradle configuration you will need to start a new project is:
+
+<embed-code file="examples/hello/build.gradle" start="plugins" end="}"></embed-code>
+```groovy
+plugins {
+    id 'io.spine.tools.gradle.bootstrap' version '1.5.24'
+}
+```
+
+Place the config into your root `build.gradle` file and execute a Gradle build. This will apply
+the Spine Bootstrap plugin to your project.
+
+You can also find this declaration on the [Gradle Plugin Portal](https://plugins.gradle.org/plugin/io.spine.tools.gradle.bootstrap),
+or on our [Getting Started page]({{site.baseurl}}/docs/quick-start).
+
+## Spine Bootstrap plugin 
+
+Spine Bootstrap plugin (Bootstrap for short) serves to automate the configuration of the modules
+in your Spine-based app.
+
+We recommend having separate Gradle subprojects for domain model definition, server  implementation,
+and client code. Bootstrap is applied a bit differently in each of these cases.
+
+### Model definitions
+
+Let's assume one of the subprojects contains Protobuf definitions for the domain model. This
+subproject is typically called `model`. In `build.gradle` for that subproject, declare:
+```groovy
+spine.assembleModel()
+```
+This way, the subproject will receive all the required Spine dependencies but no code will be
+generated from Protobuf.
```

Comment:
Let's discuss the original idea of this section tomorrow in person. I don't fully understand what you mean here :(